File: src/sdk-reference/cpp/1/auth/credentials-exist/snippets/credentials-exist.cpp
try {
  kuzzle->auth->login("local", R"({"username":"foo","password":"bar"})");
  bool exists = kuzzle->auth->credentialsExist("local");

  if (exists) {
    std::cout << "Credentials exists for local strategy" << std::endl;
  }
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/auth/credentials-exist/snippets/credentials-exist.cs
try {
  kuzzle.auth.login("local", @"{""username"":""foo"",""password"":""bar""}");
  bool exists = kuzzle.auth.credentialsExist("local");

  if (exists) {
    Console.WriteLine("Credentials exists for local strategy");
  }
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.Message());
}
